docs/Makefile

```diff
@@ -25,6 +25,23 @@ buildapi:
 	@echo
 	@echo "Auto-generation of API documentation finished. " \
 	      "The generated files are in 'apidoc/'"
+		  
```

Comment:
Could you please remove these additional make directives which point to your home directory?